application/app/src/test/java/uk/nhs/digital/uec/api/controller/FuzzySearchControllerTest.java

```diff
@@ -68,7 +68,7 @@ public void getServicesByFuzzySearchTestSucc() throws ValidationException {
     when(mockRequestParams.getFuzzLevel()).thenReturn(FUZZ_LEVEL);
     when(mockRequestParams.getMaxNumServicesToReturn()).thenReturn(MAX_SERVICES_TO_RETURN);
     when(mockFuzzyServiceSearchService.retrieveServicesByFuzzySearch(
-            SEARCH_POSTCODE, searchCriteria))
+            SEARCH_POSTCODE, searchCriteria, null))
```

Comment:
Would be better to pass through something like Collections.EmptyMap to better mimic a real world scenario rather than passing null values

Reply:
These changes were already merged, this branch was rebased from master